BuildTools/ModFile/ContentConverters.cs

```diff
@@ -0,0 +1,49 @@
+﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+using System.IO;
+
+namespace tModLoader.BuildTools.ModFile;
+
+public static class ContentConverters
+{
+	private const int Version = 1;
+
+	public static bool Convert(ref string resourceName, FileStream src, MemoryStream dst) {
+		switch (Path.GetExtension(resourceName).ToLower()) {
+			case ".png":
+				if (resourceName != "icon.png") {
+					ToRaw(src, dst);
+					resourceName = Path.ChangeExtension(resourceName, "rawimg");
+					return true;
+				}
+				src.Position = 0;
+				return false;
+			default:
+				return false;
+		}
+	}
+
+	public static void ToRaw(Stream src, Stream dst) {
```

Comment:
iirc, if I used the `ImageIO` methods in the `tModLoader` solution, I'd have to load all the libraries that tModLoader uses, which would require copying them next to the `BuildTools` dll file (for MSBuild to load them). My solution to this was to copy the method and make `BuildTools` use `ImageSharp` (to load images), making it necessary to only copy a single dll instead of many more.

Reply:
I believe ImageSharp might have some incompatibilities on Linux, even when manually providing binaries or installing them through the system package manage (at least on NixOS, which is somewhat unique). Might be a cause for concern, I can have a friend test this.